Gemfile

```diff
@@ -53,6 +53,8 @@ gem 'prawn', '< 1.0'
 gem 'prawn_rails'
 gem 'ransack'
 gem 'ri_cal'
+#gem 'roust'
+gem 'roust', :git => 'git@github.com:bulletproofnetworks/roust.git'
```

Comment:
Please use the `name: value` notation. I recently unified all this tree-wide, and we shouldn't introduce new users of the old-fashion syntax. Same counts for the actual code in this commit.